lib/taskana-core-test/src/test/java/acceptance/workbasket/get/GetWorkbasketAccTest.java

```diff
@@ -72,21 +117,9 @@ void should_ReturnWorkbasketByKeyAndDomain_When_NoExplicitPermissionButUserHasAd
   @TestTemplate
   void should_ReturnWorkbasketById_When_NoExplicitPermissionsButUserIsInAdministrativeRole()
       throws Exception {
+    Workbasket workbasket = workbasketService.getWorkbasket("user-1-2", "domain_a");
```

Comment:
Test name says "byId", but you use key and domain to get a Workbasket. Please get Workbasket by id or rename the test.